clicc_devices/urls.py

```diff
@@ -6,4 +6,5 @@
     path("logs/", views.show_log, name="show_log"),
     path("logs/<int:line_count>", views.show_log, name="show_log"),
     path("release_notes/", views.release_notes, name="release_notes"),
+    path("view_sets/", views.view_sets, name="view_sets"),
```

Comment:
Since this is the first app-specific view, let's make it the default route.  Currently, that's the entry on line 5:
`path("", views.default_view, name="default_view"),`

That can be changed to specify `view_sets` instead.  Then the explicit `view_sets/` path can be removed (or kept, if we want to support it explicitly, but I'd remove it).

That way, when the user goes to the application without specifying a path (or just the implicit `/`), they'll go to this page, and be prompted to log in if needed.

The `default_view` I provided in `views.py`, so that something would show up after creating an application from our new template, should also be removed.